### DISCUSSION TITLE
SBOM for kong debian package

### DISCUSSION THREAD

**Discussion by shusriva** [Kong/kong#11635] [Help]
Hello Team,

I am building the Kong Debian package manually from the source code due to some security constraint.
Can you please tell me how to generate software bill of materials for this Debian package (or how kong is generating sbom for their source code). I need to upload this to some tool to get the security clearance.

**Comment by pombredanne** [comment]
See also the related discussion at https://github.com/nexB/scancode-toolkit/issues/3526
